### PR TITLE
[HAC-157] Change accent color in material-ui

### PR DESCRIPTION
**Pull request by kkkowalski**
- Added HacklagTheme.js file, used to control theme settings
- TextField set to fullWidth inside LandingRegister component

### CODE REVIEW

**Comment by mkucharz** [comment]
LGTM

File: src/apps/LandingRegister/LandingRegister.js [hunk 1]
```diff
@@ -88,8 +87,8 @@ export default Radium(React.createClass({
 
             <TextField
               floatingLabelText="Your Email"
-              style={styles.emailTextfield}
               type="email"
+              fullWidth="true"
```
Comment (patrykkopycinski) [inline src/apps/LandingRegister/LandingRegister.js]: you should pass boolean not string

Comment (mkucharz) [inline src/apps/LandingRegister/LandingRegister.js]: That could work, but I agree it should be just `true`.

File: src/common/HacklagTheme.js [hunk 2]
```diff
@@ -0,0 +1,24 @@
+import Colors from 'material-ui/lib/styles/colors';
+import ColorManipulator from 'material-ui/lib/utils/color-manipulator';
+import Spacing from './Spacing';
+
+export default {
+  spacing: Spacing,
+  contentFontFamily: 'Avenir, sans-serif',
```
Comment (patrykkopycinski) [inline src/common/HacklagTheme.js:7]: did you load Avenir font into project?

Comment (mkucharz) [inline src/common/HacklagTheme.js:7]: If not it doesn't matter, because we are going to change it very soon.

Comment (patrykkopycinski) [inline src/common/HacklagTheme.js:7]: k

**Comment by mkucharz** [comment]
@kkkowalski you have to make it able to marge with devel

**Comment by kkotelczuk** [comment]
LGTM

**Pull request merged**